compiler/rustc_infer/src/infer/lexical_region_resolve/mod.rs

```diff
@@ -488,7 +488,7 @@ impl<'cx, 'tcx> LexicalResolver<'cx, 'tcx> {
                         // If this empty region is from a universe that can
                         // name the placeholder, then the placeholder is
                         // larger; otherwise, the only ancestor is `'static`.
-                        if a_ui.can_name(placeholder.universe) { true } else { false }
+                        return a_ui.can_name(placeholder.universe);
```

Comment:
```suggestion
                        a_ui.can_name(placeholder.universe)
```

Reply:
I originally skipped the return, but I thought with the return it was more obvious that we get/return a bool here instead of just "computing something"

Reply:
If you prefer keeping it I'm fine with that.